docs/utils/wait-for-all.mdx

```diff
@@ -55,13 +55,9 @@ const catsAtom = atom(async (get) => {
   return await response.json()
 })
 
-const animalsAtom = atom((get) => {
-  return get(
-    waitForAll({
-      dogs: dogsAtom,
-      cats: catsAtom,
-    })
-  )
+const animalsAtom = waitForAll({
+  dogs: dogsAtom,
+  cats: catsAtom,
```

Comment:
@Thisen Was there any intention to show the atom wrapper usage? If so, we probably want more meaningful example.

Reply:
I can't remember tbh 😅